PermissionsEx/permissions.yml

```diff
@@ -8797,6 +8797,32 @@ groups:
         - ch.alias.haste0
         - ch.alias.haste1
         - ch.alias.haste2
+    quake:
+    inheritance:
+    - commonherotraits
+    options:
+      prefix: '&8Q&7u&8a&7k&8e'
+      suffix: '&7'
+    worlds:
+      superherocity:
+        permissions:
+        - ch.alias.quake
+        - ch.alias.jump0
+        - ch.alias.jump1
+        - ch.alias.jump2
+        - ch.alias.jump3
+        - ch.alias.jump4
+        - ch.alias.jump5
+        - ch.alias.jump6
+        - ch.alias.jump7
+        - ch.alias.jump8
+        - ch.alias.jump9
```

Comment:
why so much jump?

Reply:
look at the proof

Reply:
proof of jump 9:
https://www.youtube.com/watch?v=K9oOni4Ye9s
2:51

Reply:
its at 3:00 sorry